doc/website/README.md

```diff
@@ -0,0 +1,71 @@
+---
+currentMenu: home
+---
+# Couscous Dark template
```

Comment:
Copied until https://github.com/CouscousPHP/Template-Dark/pull/5 is merged. Then we will remove custom theme.